### PR TITLE
e2e-script

### PR DESCRIPTION
**Pull request by dennis00010011b**
### e2e script for bridge-ui
Configure startURL, homeAccount, foreignAccount in ```config.json```
#### Tests

```
1. User is able to open main page of bridge-ui 
2. Main page: foreign POA balance is displayed 
3. Main page: home POA balance is displayed

4. User is able to send tokens from Home account to Foreign account
5. Home POA balance has  correctly changed after transaction
6. Foreign account has received correct amount of tokens after transaction

7. User is able to send tokens from Foreign account to Home account
8. Foreign POA balance has correctly changed after transaction
9. Home account has received correct amount of tokens after transaction

```

### CODE REVIEW

File: e2e-script/MetaMask.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    await this.switchToNextPage();
+  }
+
+  async clickImportAccount() {
+    await  super.clickWithWait(popupAccount);
+    await this.driver.executeScript( "document.getElementsByClassName('dropdown-menu-item')["+(accountOrderNumber+1)+"].click();");
+    accountOrderNumber++;
+  }
+
+  async doTransaction(refreshCount) {
```
Comment (rstormsf) [inline e2e-script/MetaMask.js]: refactor this function. eleminate if conditions, remove while(true)

File: e2e-script/MetaMask.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+       url="https://sokol.poa.network";
+       networks.push(77);
+       break;
+      }
+      case 99: {
+        url="https://core.poa.network";
+        networks.push(99);
+        break;
+      }
+      case 7762959: {
```
Comment (rstormsf) [inline e2e-script/MetaMask.js]: really?

File: e2e-script/MetaMask.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      	url="https://sokol.poa.network";break;
+      } //Musicoin=>SOKOL
+      default: {
+	      url="https://sokol.poa.network";
+      }
+    }
+    await this.driver.executeScript("document.getElementsByClassName('dropdown-menu-item')["+
+	                                 (networks.length-1)+"].click();");
+    await this.driver.sleep(3000);
```
Comment (rstormsf) [inline e2e-script/MetaMask.js]: why sleep ?

File: e2e-script/test.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
+let test = require('selenium-webdriver/testing');
+let assert = require('assert');
+const fs = require('fs-extra');
+const utils=require('./Utils.js');
+const Utils=utils.Utils;
+const mtMask=require('./MetaMask.js');
+const MetaMask=mtMask.MetaMask;
```
Comment (rstormsf) [inline e2e-script/test.js]: 2 metamasks? can u elaborate?

**Comment by rstormsf** [comment]
@patitonar please let us know if it's ok to merge it in

**Comment by patitonar** [comment]
@rstormsf looks good to me. We will need to update the tests with the new confirmation alert on transfer click #43

**Pull request merged**